src/gt4py/backend/gtc_backend/common.py

```diff
@@ -305,10 +306,11 @@ def make_extension(
             stencil_ir = self.builder.gtir
         # Generate source
         if not self.builder.options._impl_opts.get("disable-code-generation", False):
-            gt_pyext_sources: Dict[str, Any] = self.make_extension_sources(stencil_ir=stencil_ir)
-            gt_pyext_sources = {**gt_pyext_sources["computation"], **gt_pyext_sources["bindings"]}
+            gt_pyext_files: Dict[str, Any] = self.make_extension_sources(stencil_ir=stencil_ir)
+            gt_pyext_sources = {**gt_pyext_files["computation"], **gt_pyext_files["bindings"]}
         else:
             # Pass NOTHING to the self.builder means try to reuse the source code files
+            gt_pyext_files = {}
```

Comment:
Does this need a type hint too? Or, put one above line 307.

Reply:
I put it above